MyApp/app/controllers/users/registrations_controller.rb

```diff
@@ -2,27 +2,36 @@
 
 class Users::RegistrationsController < Devise::RegistrationsController
   # before_action :configure_sign_up_params, only: [:create]
-  # before_action :configure_account_update_params, only: [:update]
+  before_action :configure_permitted_parameters, only: [:update]
 
   # GET /resource/sign_up
-  # def new
-  #   super
-  # end
+  def new
+    super
+  end
 
   # POST /resource
-  # def create
-  #   super
-  # end
+ def create
+    super
+    users = User.all
+    if users.length == 1
+      @user.add_role(:admin)
+    else
+      @user.add_role(:author)
+    end
```

Comment:
It's fine, but you can make it shorter:
User.all.length == 1 ? @user.add_role(:admin) : @user.add_role(:author)